src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.addEventListener('click', (e) => {
+  if (e.target.tagName !== 'TH') {
+    return;
+  }
+
+  const index = e.target.cellIndex;
+  const type = e.target.innerText;
+  const tBody = document.querySelector('tbody');
+  const tr = [...tBody.querySelectorAll('tr')];
```

Comment:
it is better to give domain related names - tBody => table, tr => employees (or employeeRows)